packages/forms/src/UIForm/fields/Button/SingleButton.component.js

```diff
@@ -46,6 +46,6 @@ if (process.env.NODE_ENV !== 'production') {
 			title: PropTypes.string,
 			triggers: PropTypes.arrayOf(PropTypes.string),
 			type: PropTypes.string,
-		}),
+		}).isRequired,
```

Comment:
schema is required and none of its component is, isn't it more safe then to rely on a default props set to an empty literral object ?